Gemfile.lock

```diff
@@ -11,7 +11,7 @@ GEM
     colored2 (3.1.2)
     cork (0.3.0)
       colored2 (~> 3.1)
-    danger (8.5.0)
+    danger (8.6.1)
```

Comment:
Do we need to update all of these or is there a specific dependency we can just update?

Reply:
Octokit is the main one but I usually update everything when I do this stuff.

Reply:
OK. If that's what you've already been doing.